docs/experimental_features/modin_in_the_cloud.rst

```diff
@@ -0,0 +1,68 @@
+Modin in the Cloud
+==================
+
+The modin implements functionality that allows to transfer computing to the cloud.
+Please note that this feature is experimental and behavior or interfaces could be changed.
+
+Prerequisites
+"""""""""""""
+
+  * Sign up with a cloud provider and get credentials file. Currently supported only AWS.
+  (For AWS, on credentials file format see https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where)
+
+Setup environment
+"""""""""""""""""
+pip install modin[remote] - which install the following dependencies:
+  * RPyC - https://github.com/tomerfiliba-org/rpyc
+  Allows to perform remote procedure calls.
+  * Cloudpickle - https://github.com/cloudpipe/cloudpickle
+  Allows pickling of functions and classes, which is used in our distributed runtime.
+  * Boto3 - https://github.com/boto/boto3
+  Allows to create and setup AWS cloud machines. Optional library for Ray Autoscaler.
```

Comment:
please revise this section formatting, it looks very wrong when rendered, see https://modin--3307.org.readthedocs.build/en/3307/experimental_features/modin_in_the_cloud.html#setup-environment